english/marshmallow.md

```diff
@@ -2,6 +2,10 @@ Your adventure begins. You realize there are hundreds of paths for you to choose
 Which way should you go? Only you can decide. 
 
 
+January 2017 approaches. You have two presidential candidates to vote for.
+[Click](vote-the-president/Trump.md) here to vote for Donald Trump.
+[Click](vote-the-president/Hillary.md) here to vote for Hillary Clinton.
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.